docs/developer-docs/integrations/sns/managing-sns/sns-asset-canister.md

```diff
@@ -0,0 +1,146 @@
+# SNS asset canisters
+
+## Overview
+
+The [asset canister](https://github.com/dfinity/sdk/tree/master/src/canisters/frontend/ic-frontend-canister) provides users with a way to store and retrieve static assets from a canister deployed on the IC. Generally, asset canisters are used to serve HTML, CSS, or JavaScript assets, which are typically part of a dapp's frontend. For this reason, the asset canister is also referred to as the frontend canister. For purposes of this guide, we'll refer to it as the asset canister. 
+
+In the context of the SNS, a dapp's associated asset canister serves the frontend assets related to dapp and may also include a frontend to the SNS DAO, e.g., through which users can vote on governance proposals.  
+
+The contents of the asset canister must be configured prior to the launch of the SNS, and any changes afterwards must be made by a principal with the permission to do so. The SNS is the only one that can update the list of approved principals that have this permission, and any additions to this list approved on via an SNS proposal. This configuration assures that changes to the asset canister are only made by authorized principals that the SNS has agreed on through proposals. These changes are referred to as 'upgrades' to the asset canister in the remainder of this document. 
+
+When a dapp's control is handed over to an SNS, this is also true for the associated asset canister.
+
+This section is relevant if your project contains an asset canister and describes how you can test handing over control of an asset canister to an SNS.
+
```

Comment:
Would it make sense to have here a high level introduction how all of this works? 
I imagine the following content:
* what are the roles that can be set in the asset canister and what can these roles do?
* In a non-DAO setting who would have these roles? 
* Once asset canister is handed over to SNS DAO, who has which roles and can do what?